main.go

```diff
@@ -17,45 +17,51 @@ import (
 const sqlBoilerVersion = "2.6.0"
 
 var (
+	cfgFile   string
```

Comment:
For global vars (usually flags) I prefer to keep them separated by their type in some way, usually I'd call this `flagConfigFile`, there's no reason to abbreviate to `cfg` either.